integration-tests/.vscode/launch.json

```diff
@@ -8,13 +8,9 @@
             "type": "node",
             "request": "launch",
             "name": "Launch Program",
-            "skipFiles": [
```

Comment:
Everything within `.vscode` should be ignored since these are VS Code generated files.

Reply:
Excluded the `.vscode` folder